multigeiger/multigeiger.ino

```diff
@@ -754,25 +751,21 @@ void DisplayStartscreen(void) {
 
   u8x8.clear();
   #if CPU == STICK
-  // Display is only 4 lines by 8 characters; lines counting from 2 to 5
-  u8x8.setFont(u8x8_font_5x8_f);        // use really small font
-  for (int i = 2; i < 6; i++) {
-    u8x8.drawString(0, i, "        ");  // clear all 4 lines
-  }
+  // Display is only 4 lines by 8 characters; lines counting from 2 to 5.
```

Comment:
the docs say X and Y both start from 0.

so, the docs are wrong? or do we want to leave 2 empty lines at the top?

Reply:
Again: file a bug against the library? It's docs say 0/0.